src/main/java/io/confluent/kafkarest/ProducerPool.java

```diff
@@ -143,7 +144,8 @@ private static String getBootstrapBrokers(ZkUtils zkUtils) {
         if (bootstrapBrokers.length() > 0) {
           bootstrapBrokers += ",";
         }
-        bootstrapBrokers += ep.connectionString();
+        String hostport = ep.host() == null ? ":" + ep.port() : Utils.formatAddress(ep.host(), ep.port());
+        bootstrapBrokers += ep.securityProtocol() + "://" + hostport;
```

Comment:
It works with and without as discussed in https://github.com/confluentinc/schema-registry/pull/474/files/2c3cf54751e0991140529859096c1f3fc6938719#diff-949e07df8a5fc0851347cd6e6a971a84 .